packages/db/config.d.ts

```diff
@@ -0,0 +1,549 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type CrudOperationAuth =
+  | {
+      /**
+       * checks for the operation
+       */
+      checks?: {
+        [k: string]: {
+          [k: string]: unknown;
+        };
+      };
+      /**
+       * array of enabled field for the operation
+       */
+      fields?: string[];
+    }
+  | boolean;
+
+export interface PlatformaticDB {
+  server: {
```

Comment:
Is it possible to reexport server options from the fastify package?

Reply:
Unfortunately no :(